modules/ggl-http/src/aws_sigv4.h

```diff
@@ -0,0 +1,62 @@
+// aws-greengrass-lite - AWS IoT Greengrass runtime for constrained devices
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef GGHTTPLIB_AWS_SIGV4_H
+#define GGHTTPLIB_AWS_SIGV4_H
+
+#include "ggl/http.h"
+#include <ggl/buffer.h>
+#include <ggl/error.h>
+#include <ggl/vector.h>
+
+#define ZERO_PAYLOAD_SHA \
+    "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+
+#define HTTP_METHOD_GET "GET" /**< HTTP Method GET string. */
+#define HTTP_METHOD_PUT "PUT" /**< HTTP Method PUT string. */
+#define HTTP_METHOD_POST "POST" /**< HTTP Method POST string. */
+#define HTTP_METHOD_HEAD "HEAD" /**< HTTP Method HEAD string. */
+
+#define X_AMZ_SECURITY_TOKEN "x-amz-security-token"
+#define X_AMZ_DATE "x-amz-date"
+#define X_AMZ_CONTENT_SHA256 "x-amz-content-sha256"
+#define HOST_HEADER "host"
```

Comment:
These macros don't seem very useful. They are constants with meaning that the macro name doesnt add to. Would be better to just use the string literals